config/kubernetes/sig-release/teams.yaml

```diff
@@ -250,19 +250,23 @@ teams:
         - bai # 1.20 RT Bug Triage Lead
         - claurence # subproject owner
         - cpanato # Release Manager
+        - eagleusb # 1.20 RT Docs Shadow
         - guineveresaenger # subproject owner
         - hasheddan # Release Manager / 1.20 RT Lead Shadow
         - jameslaverack # 1.20 RT Release Notes Lead
         - jeremyrickard # 1.20 RT Lead
         - jrsapi # 1.20 Communications Lead
         - justaugustus # subproject owner
         - kikisdeliveryservice # 1.20 Enhancements Lead
+        - kcmartin # 1.20 RT Docs Shadow
```

Comment:
Ah I think this needs to be moved up one :)

Reply:
Good catch! Fixed it, thank you for always double-checking! :D